### PR TITLE
Feat: Add getErrors public method on FlatfileRecord class

### PR DESCRIPTION
**Pull request by mattheri**
## Please explain how to summarize this PR for the Changelog:

Added public `getErrors` method on the FlatfileRecord class.

## Tell code reviewer how and what to test:

Add an error to a field. Then call `getErrors` to see the error returned.

### CODE REVIEW

File: packages/hooks/src/classes/FlatfileRecord.ts
Comment (dboskovic) [inline packages/hooks/src/classes/FlatfileRecord.ts:235]: Could we add an optional parameter to pass the key here?

Comment (mattheri) [reply packages/hooks/src/classes/FlatfileRecord.ts]: I have added a `fields` argument which can take one `keyof M` or an array of `keyof M`.

**Comment by mattheri** [comment]
@coderabbitai review

**Review by dboskovic — APPROVED** [review]

**Comment by mattheri** [comment]
@dboskovic there are still 2 checks pending approval. Shall I wait for someone else to approve those?

**Review by bangarang — DISMISSED** [review]

**Review by bangarang — APPROVED** [review]

**Pull request merged**
